microsoft-edge/privacy-whitepaper/index.md

```diff
@@ -747,19 +747,39 @@ To enable diagnostic data collection, go to `edge://settings/privacy`. Turn on t
 
-If you select **Basic** under the **Use writing assistance** toggle in `edge://settings/languages`, the service will perform only local spellchecking on the device, and no data will be sent to the cloud for spellchecking services. 
+If you select **Basic** instead of **Microsoft Editor**, Microsoft Edge will perform only local spellchecking on the device, and no data will be sent to the cloud for spellchecking services.
 
-To turn off all spelling and grammar capabilities, turn off the **Use writing assistance** toggle.
+To use **Basic** spelling and grammar checking: 
+1. Go to `edge://settings/languages`.
+1. In the **Use writing assistance** section, select **Basic**.
+
+To turn off all spelling and grammar capabilities:
+1. Go to `edge://settings/languages`.
+1. Turn off the **Use writing assistance** toggle.
+
+#### Languages
+
+When you add a new language to Microsoft Edge, the browser downloads the dictionary for the new language to the device over HTTPS. The dictionary is used for the basic spellcheck service or for languages that aren't supported by Microsoft Editor spelling and grammar checking. Deleting the language from your Microsoft Edge settings deletes the dictionary from the device. 
  
-When you add a new language to Microsoft Edge, the browser downloads the dictionary for the new language to the device over HTTPS. The dictionary is used for the basic spellcheck service or for languages that aren't supported by Microsoft Editor spelling and grammar checking. Deleting the language from your Microsoft Edge settings deletes the dictionary from the device.
+#### Text prediction 
+
+Microsoft Edge automatically provides word and sentence predictions in certain text boxes on the web. These predictions are only visible to you and are not inserted into the text box until you press `Tab` or the `Right arrow` key to accept them.
+
+To turn off text predictions in Microsoft Edge: 
+
+1. Go to `edge://settings/languages`.
+
+2. Turn off the **Use text prediction** toggle.
+
+If the **Use text prediction** toggle is turned on, Microsoft Edge sends the text in the text box, your top language from the browser setting, and a pseudonymous text box identifier to a Microsoft cloud service over a secure HTTPS connection. The Microsoft cloud service processes the text to generate a relevant text prediction. Typed characters and text predictions are cached for up to 30 days, for service quality and performance improvement purposes only.  
```

Comment:
do we use "pseudonymous" elsewhere? I'm not sure what this means in this context? 

maybe, "a text box identifier (not associated with your account)" ?